mltsp/featurize.py

```diff
@@ -226,9 +226,9 @@ def featurize_time_series(times, values, errors=None, features_to_use=[],
 
     if labels is None:
         if isinstance(times, (list, tuple)):
-            labels = np.arange(len(times)).astype('str')
```

Comment:
This change isn't important, I just changed my mind about having the labels default to strings since this makes sorting a pain; no reason we can't use ints or anything else we want as labels.